RealmSwift/Sync.swift

```diff
@@ -536,7 +536,7 @@ public extension User {
 
     /**
      The custom data of the user.
-     This is configured in your MongoDB Realm App.
+     This is configured in your Atlas Device Sync app.
```

Comment:
In this docs, Device Sync is the feature name but Atlas App Services app would be where the configuration is managed.
```suggestion
     This is configured in your Atlas App Services app.
```